examples/hitl/rearrange_v2/config/language_rearrange_multi_agent_llm_gui.yaml

```diff
@@ -0,0 +1,153 @@
+# This config is derived from habitat-lab/habitat/config/benchmark/multi_agent/hssd_spot_human.yaml
+# @package _global_
+
+defaults:
+  - /habitat: habitat_config_base
+  # - /habitat/task: task_config_base
+
+  # Human setup
+  - /habitat_conf/habitat_agent@habitat.simulator.agents.agent_1: rgbd_head_rgbd_arm_agent_vis
+  - /habitat/simulator/agents@habitat.simulator.agents.agent_1: human
+  # following are from hab-llm entry config. Questions:
+  # 1. What is the difference between
+  #    habitat.task.actions.agent_1_humanoid_base_velocity and
+  #    habitat.task.actions.agent_1_base_velocity?
+  # 2. What is the humanoidjoint_action? Do we need it here?
+  # - /habitat/task/actions@habitat.task.actions.agent_1_humanoid_base_velocity: humanoid_base_velocity
+  # - /habitat/task/actions@habitat.task.actions.agent_1_humanoidjoint_action: humanoidjoint_action
+  # - /habitat/task/actions@habitat.task.actions.agent_1_base_velocity: base_velocity
```

Comment:
@xavierpuigf Would you know the answers to these?

Reply:
1. you don't need agent_1_base_velocity for the human, only agent_1_humanoid_base_velocity. The difference is that humanoid base velocity generates a human realistic motion whereas base velocity only moves the base (used to move the robot for instance).
2. humanoidjoint_action sets the human joints so that the pose is realistic. You will never be callign it yourself, the agent_1_base_velocity sets that. But you sitll need to define it here